<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayControl</class>
 <widget class="QWidget" name="PlayControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>播放控制</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#PlayControl
{
	border-radius: 5px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,5,0,0,0,0,1,0,0">
   <item>
    <widget class="QLabel" name="labelIcon">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../res/resources.qrc">:/icons/icons/mxlplayer_icon256.ico</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPlayTime">
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSliderPosition">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTotalTime">
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonPlayPause">
     <property name="toolTip">
      <string>播放/暂停</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/resources.qrc">
       <normaloff>:/images/images/play.png</normaloff>:/images/images/play.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonMute">
     <property name="toolTip">
      <string>静音</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/resources.qrc">
       <normaloff>:/images/images/volume.png</normaloff>:/images/images/volume.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSliderVolume">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonBack">
     <property name="toolTip">
      <string>退出</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/resources.qrc">
       <normaloff>:/images/images/close.png</normaloff>:/images/images/close.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
